docker-compose.yml

```diff
@@ -57,6 +57,31 @@ services:
       - "traefik.http.routers.backend-secured.rule=Host(`api.${HOST_NAME}`)"
       - "traefik.http.routers.backend-secured.tls.certresolver=mytlschallenge"
       - "traefik.docker.network=traefik"
+  rust_backend:
+    build:
+      context: ./rust_backend
+      target: ${APP_ENVIRONMENT}
+      args:
+        - APP_ENV=${APP_ENVIRONMENT}
+    container_name: rust_backend
+    restart: always
+    ports:
+      - 9000:9000
+    links:
+      - redis
+    networks:
+      - traefik
+      - streamchaser
+    volumes:
+      - ./rust_backend:/app
+    depends_on:
+      - traefik
```

Comment:
Add redis here if you remove the `links:`

Reply:
Done